src/MahApps.Metro.Samples/MahApps.Metro.Demo/ExampleViews/ProgressBars.xaml

```diff
@@ -12,7 +12,7 @@
 
     <UserControl.Resources>
         <ResourceDictionary>
-            <Style BasedOn="{StaticResource MahApps.Metro.Styles.MetroHeader}" TargetType="{x:Type Controls:MetroHeader}">
+            <Style BasedOn="{StaticResource MahApps.Styles.MetroHeader.Metro}" TargetType="{x:Type Controls:MetroHeader}">
```

Comment:
Maybe some controls should be renamed as part of this breaking change version (probably not in this commit). It seems a bit weird to have `MahApps.Styles.MetroHeader.Metro`